app/controllers/posts_controller.rb

```diff
@@ -9,6 +9,12 @@ def index
 
   def show
     @comment_pagy, @comments = pagy(@post.comments.where(parent_id: nil), items: 5)
+
+    return unless user_signed_in? && current_user == @post.user
+
+    @post.comments.each do |comment|
```

Comment:
use update_all method

Reply:
rubocop offense avoid using update_all because it skips validations.

Reply:
disable it at all